docs/data-sources/resource_policy.md

```diff
@@ -0,0 +1,134 @@
+# Data Source: mongodbatlas_resource_policy
+
+`mongodbatlas_resource_policy` describes a resource policy in an organization.
+
+-> **NOTE**: Resource Policies are currently in Public Preview. You will need to
```

Comment:
same change as above

Reply:
Changed in a8851acdb8c44933b67c874a3160f5b6938a5ba2